test/jasmine/karma.conf.js

```diff
@@ -113,6 +116,13 @@ if(isSingleSuiteRun) {
     func.defaultConfig.preprocessors[pathToMain] = ['browserify'];
     func.defaultConfig.preprocessors[testFileGlob] = ['browserify'];
 }
+else if(isRequireJSTest) {
+    func.defaultConfig.files = [
+        constants.pathToRequireJS,
+        constants.pathToRequireJSFixture,
+        testFileGlob
+    ];
```

Comment:
Note: don't browserify the `bundle_tests/requirejs_test.js` suite as browserify and require.js don't get along.